snooty/main.py

```diff
@@ -3,7 +3,7 @@
 Usage:
   snooty build [--no-caching] <source-path> [--output=<path>] [options]
   snooty watch [--no-caching] <source-path> [options]
-  snooty create-cache         <source-path> [options]
+  snooty create-cache [--no-caching]         <source-path> [options]
```

Comment:
The space alignment here is just a little odd 😅 I think either all the `<source-path>`'s should be lined up, or this line should just have a single space